### ISSUE TITLE
Report ERR_CHILD_PROCESS_STDIO_MAXBUFFER stdout maxBuffer length exceeded

### ISSUE THREAD

**Issue by jkyin** [vknabel/vscode-swiftlint#3]
`RangeError [ERR_CHILD_PROCESS_STDIO_MAXBUFFER]: stdout maxBuffer length exceeded
	at Socket.onChildStdout (child_process.js:363:14)
	at Socket.emit (events.js:200:13)
	at addChunk (_stream_readable.js:294:12)
	at readableAddChunk (_stream_readable.js:271:13)
	at Socket.Readable.push (_stream_readable.js:210:10)
	at Pipe.onStreamRead (internal/stream_base_commons.js:166:17)`

**Comment by vknabel** [comment]
Hi @jkyin, thanks for your report. It seems like SwiftLint had lots of warnings or a huge stack trace in your repo!
I just released [vscode-swiftlint@1.0.2](https://github.com/vknabel/vscode-swiftlint/releases/tag/1.0.2) which should solve your issue. Though I did only increase the buffer size from 200 KiB to 500 KiB. I hope this is enough!

In case it's not, try running swiftlint manually on the file using the command line.
